pkg/database/statistics.go

```diff
@@ -104,7 +138,7 @@ func (u *User) GetTotals(t WorkoutType) (*Bucket, error) {
 		Table("workouts").
 		Select(
 			"count(*) as workouts",
-			"type as workout_type",
+			"max(type) as workout_type",
```

Comment:
The `max` function is useless around `type`, since `type` is part of the `where` and expected to be a single (distinct) value: we search for all totals for a given type
Eg. what is the sum total duration for running?

Reply:
postgres is very strict about requiring all selected fields to either be in an aggregate function or in the `group by`  clause. So even though only a single `type` will be returned it still needs to be wrapped in an aggregate function.

Reply:
TIL! Thanks.